### PR TITLE
EUAA min marks per lsit

### PR DESCRIPTION
**Pull request by av-mads**

### CODE REVIEW

File: test/selection_pile_validator.test.ts [hunk 1]
```diff
@@ -176,4 +177,20 @@ describe("validate", () => {
       expect(validator.validate(selectionPile)[0].message).to.equal("exceeded_list_limit")
     });
   })
+
+  context("with lazy errors", ()=> {
+    context("with below list limit", () => {
```
Comment (av-martin) [inline test/selection_pile_validator.test.ts:182]: This doesn't appear to test lazyness?

Comment (av-mads) [reply test/selection_pile_validator.test.ts:182]: It was just to have all lazy errors tests under a scope.

Comment (av-mads) [reply test/selection_pile_validator.test.ts:182]: below list limit is a lazy error

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "4.0.1",
+  "version": "4.0.2",
```
Comment (av-martin) [inline package.json:2]: Technically this should be a minor version change, since you added new functionality

**Review by av-martin — APPROVED** [review]

**Pull request merged**